client/my-sites/marketing/do-it-for-me/difm-landing.tsx

```diff
@@ -238,16 +232,24 @@ export default function DIFMLanding( {
 	const product = useSelector( ( state ) => getProductBySlug( state, WPCOM_DIFM_LITE ) );
 	const productCost = product?.cost;
 
+	const planObject = getPlan( PLAN_PREMIUM );
+	const planTitle = planObject?.getTitle();
```

Comment:
`Non-blocking / Nit:`

I didn't see this explicitly state in the P2 discussion, but it looks like we're no longer considering the `PLAN_WPCOM_PRO` when generating FAQ text. Just wanted to confirm this is intentional 🙂

Reply:
Thanks for the review, Jeremy! Yes, the Pro plan was removed long ago, but somehow this check was still in the code base.